### DISCUSSION TITLE
typography plugin vs mdx components

### DISCUSSION THREAD

**Discussion by stefanprobst** [tailwindlabs/tailwindcss#4064] [Help] [answered]
i'm using `@tailwindcss/typography` to style content in an `.mdx` file. however, when using custom JSX components in mdx, I generally don't want the `prose` styles to affect the component styles (especially since the mdx compiler will wrap block elements in `<p>`). example:

```md
# Heading

Some text

<Admonition title="Note">

Markdown **inside** jsx! I want to opt out of styling the markup of this component with `prose`.

</Admonition>
```

i realize this is just how the cascade works, but i'm curious how others have dealt with this? thanks!

**Comment by simonswiss** [comment, marked as answer]
Hey!

If you're using the new `jit` mode, one t thing you could do is use the built in `important` modifier:

https://tailwindcss.com/docs/just-in-time-mode#built-in-important-modifier

That gives you a way to undo/override values applied by the `prose` plugin.
